users-profile-sample-application.md

```diff
@@ -0,0 +1,177 @@
+---
+id: users-profile-sample-application
+title: Users Profile Sample Application
+sidebar_label: Gin-Mongo Sample 2
+tags:
+  - quickstart
+  - go
+  - sdk
+  - tutorial
+---
+
+# Users-Profile
+
+A sample application that get, create, update, and delete the data of a user in the database (MongoDB for this application).
+
+## Prerequisites
+
+1. [Go](https://go.dev/doc/install) 1.16 or later
+2. [Docker](https://docs.docker.com/engine/install/) for running Keploy server
+3. [Thunder Client](https://marketplace.visualstudio.com/items?itemName=rangav.vscode-thunder-client) / [Postman Desktop Agent](https://www.postman.com/downloads/postman-agent/) for testing localhost APIs
+4. Code Editor ([VSCode](https://code.visualstudio.com/download), [Sublime Text](https://www.sublimetext.com/download), etc.)
+
+## Installation
+
+### Start Keploy Server
+
+```
+git clone https://github.com/keploy/keploy.git
+cd keploy
+docker-compose up
+```
+
+### Start Users-Profile sample application
+
+```
+git clone https://github.com/keploy/samples-go
+cd samples-go
+cd users-profile
+go get .
+go run .
+```
+
+## Routes
+
+> Sample Application Port: http://localhost:8080
+
+- `/user` : POST - Create a new user in the database
+- `/user/:userId` : GET - Get a user from the database
+- `/user/:userId` : PUT - Edit an existing user in the database
+- `/user/:userId` : DELETE - Delete an existing user from the database
+- `/users` : GET - Get all users from the database
+
+## Generate Test Cases
+
+> Keploy Port: http://localhost:8081/testlist
```

Comment:
@saksham140023 Can you please reconsider [this](https://github.com/keploy/docs/pull/64/files#r986723410) review?